code/modules/randomMaps/vault_definitions.dm

```diff
@@ -13,11 +13,17 @@ var/list/existing_vaults = list()
 
 /datum/vault/proc/initialize(list/objects)
 	existing_vaults.Add(src)
-
+	
 	for(var/turf/new_turf in objects)
-		if(!src.location) location = new_turf
-
+		if(!location) location = new_turf
 		new_turf.flags |= NO_MINIMAP //Makes the spawned turfs invisible on minimaps
```

Comment:
I put another "new_turf.flags |= NO_MINIMAP" below after the ChangeTurf

but if you think it's too shitty tell me and I'll do it your way